.github/workflows/vulnerability_scan_subworkflow.yml

```diff
@@ -82,40 +41,46 @@ jobs:
       - name: Install xmllint
         uses: ./.github/actions/install-xmllint
 
-      - name: Get EE dist ZIP
+      - name: Get ${{ matrix.image.label }} dist ZIP
         run: |
-          . .github/scripts/ee-build.functions.sh
-          HZ_VERSION=$(awk -F '=' '/^ARG HZ_VERSION=/ {print $2}' hazelcast-enterprise/Dockerfile)          
-          HAZELCAST_EE_ZIP_URL=$(get_hz_dist_zip "" "${HZ_VERSION}")        
-          curl --fail --silent --show-error --location "$HAZELCAST_EE_ZIP_URL" --output hazelcast-enterprise/hazelcast-enterprise-distribution.zip;
+          # Make a dummy empty ZIP file to avoid scanning Java dependencies, as managed downstream
+          # DI-50 - Remove java artifacts scanning from hazelcast-docker
+          working_directory=hazelcast-distribution
+          mkdir -p ${working_directory}/lib
+          mkdir -p ${working_directory}/bin
+          touch ${working_directory}/bin/empty
```

Comment:
nit 
you can mkdir & touch in one cmd i think `install -D /dev/null ${working_directory}/bin/empty`

Reply:
That doesn't work for me (on OSX at least)